TextFieldEffects/TextFieldsDemo/ExampleTableViewController.swift

```diff
@@ -22,8 +22,10 @@ class ExampleTableViewController : UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        _ = textFields.forEach { $0.textAlignment = .center }
```

Comment:
You can use a boolean like just a few lines below. Essentially you are changing the example project to make them all center